README.md

```diff
@@ -1,4 +1,141 @@
+
++ Move `riot.mount` to the `$.get()` callback within `loadMessages()` (in `client.js`) to ensure the riot element is mounted when there is data
+
+If you're worried `riot` is not defined in your `client.js`, don't be as it's a global variable.    
+[JSHint](http://jshint.com/) however, _will_ complain about this.    
+
+![must-add-riot-to-jshint-globals](https://cloud.githubusercontent.com/assets/4185328/9226392/5e0bf338-4106-11e5-85ca-172a14328d1d.png)
+
++ To fix this, add 'riot' to globals in your `.jshintrc` file
+
+
+### 6. [Get data from Redis to be rendered using Riot](https://github.com/dwyl/hapi-socketio-redis-riot-chat-example/tree/b755c52189805c0a402753cefecc2ae069a21383)
+**Every time a new message is input** through the message input box, we want to **save it to an array** (`messageStore`). Changes made to this array will be available to `<message>`
++ Create empty array to hold new messages (`messageStore`)
+
+renderMessage() is called once for every message (both from Redis and from Socket.io), so this is the ideal place to update our `messageStore` array.
+
++ Add messages to our array using push in `renderMessage()`
++ Pass the `messageStore` array to our riot `tag` as an option in riot.mount: `riot.mount('message', {messageStore: messageStore});`
++ `messageStore` is now available to be used in `<message>` and you can now replace the dummy data with the live data from this array using `this.messageStore = opts.messageStore`
++ Switch your data source from `dummyData` to `messageStore` in the riot `each` loop
+_Note: Any messages input into the message input box at this stage and going into the database and into our `messageStore` array, but are not yet being rendered on the page by riot - if you refresh your page however, messages are loaded in from Redis and will appear in the browser_.
+
+
+### 7. [Get riot to render new messages to the page](https://github.com/dwyl/hapi-socketio-redis-riot-chat-example/tree/a71edd3da901ef6b01f5206578ffbb26aa810bdd)
+Everything is set up for Riot to function correctly, we just need to tell it when changes are made to `messageStore` so that it can render these to the browser.
+
++ Add `riot.update()` to `renderMessage()` to be called after updating the `messageStore` array.
+
+:tada:  :tada:  :tada:  :tada: 
+##Bug fixes
+### 8. [scrollToBottom bug](https://github.com/dwyl/hapi-socketio-redis-riot-chat-example/tree/b85121e0aa14c803d4041dd730b8189c86feab5f)
+It looks like someone has raised an issue on our repo!
+https://github.com/dwyl/hapi-socketio-redis-riot-chat-example/issues/3
+
+**Why is this happening?** 
++ Try adding `console.log($('#messages').height())` to `scrollToBottom()` and refresh your page to see what happens
+
+We're calling `scrollToBottom()` straight after calling `riot.mount` which means **Riot has not yet had the time to update the page**. What we want to do is call it right after `riot.mount` **_finishes_**. To fix this:
```

Comment:
Worth making this explanation more explicit?

...to update the page and there is nothing to scroll to the botom of!